bazel/markdown_to_html/src/index.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {marked} from 'marked';
+import {hooks} from './hooks';
+import {readFile} from 'fs/promises';
+
+export async function parseMarkdown(filepath: string): Promise<string> {
+  marked.use({
+    hooks,
+    // Marked will return a promise if the async option is true.
+    // The async option will tell marked to await any walkTokens functions before parsing the tokens and returning an HTML string.
+    // More details: https://marked.js.org/using_pro#async
```

Comment:
Expand this comment to call out explicitly why this setting matters for our use of marked?